tests/Resources/Page/ODH/AiApps/Rhosak.robot

```diff
@@ -0,0 +1,49 @@
+*** Settings ***
+Library         SeleniumLibrary
+Library         ../../../../libs/Helpers.py
+Resource        ../JupyterHub/JupyterLabLauncher.robot
+Resource        ../../Components/Components.resource
+
+
+*** Keywords ***
+Check Consumer and Producer Output Equality
+  [Arguments]  ${producer_text}  ${consumer_text}
+  ${producer_output_list}=  Text To List  text=${producer_text}
+  ${consumer_output_list}=  Text To List  text=${consumer_text}
+  Should Be Equal    ${producer_output_list}    ${consumer_output_list}[1:]
+
+Load Kafka Notebooks From Git
+  [Arguments]  ${git_repo_http}
+  Open With JupyterLab Menu  File  New  Notebook
+  Sleep  1
+  Maybe Select Kernel
+  Sleep  3
+  Close Other JupyterLab Tabs
+  Clone Git Repository  REPO_URL=${git_repo_http}
+  Sleep  5
```

Comment:
Why not use the keywords we already have to clone from Git?

Reply:
"Clone Git Repository" is one of the already developed. But actually, I think I could remove the lines from 17 to 20 and remove this "Load Kafka Notebooks From Git". 
Thanks